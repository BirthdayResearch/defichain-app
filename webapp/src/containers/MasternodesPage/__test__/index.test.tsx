import React from 'react';
import Masternode from '../index';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import store from '../../../app/rootStore';

describe('Masternode Chart', () => {
  it('should check for snapshot', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <Masternode />
      </Provider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
