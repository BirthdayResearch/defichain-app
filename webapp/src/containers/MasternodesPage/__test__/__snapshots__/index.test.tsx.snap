// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Chart should check for snapshot 1`] = `
<div
  className="main-wrapper"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Masternodes â€“ DeFi Blockchain Client
    </title>
  </HelmetWrapper>
  <header
    className="header-bar"
  >
    <h1
      className=""
    >
      Masternodes
    </h1>
    <ButtonGroup
      className=""
      role="group"
      tag="div"
    >
      <Button
        color="link"
        onClick={[Function]}
        size="sm"
        tag="button"
      >
        <MdSearch />
      </Button>
      <Button
        color="link"
        tag={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "NavLink",
            "propTypes": Object {
              "activeClassName": [Function],
              "activeStyle": [Function],
              "aria-current": [Function],
              "className": [Function],
              "exact": [Function],
              "innerRef": [Function],
              "isActive": [Function],
              "location": [Function],
              "onClick": [Function],
              "replace": [Function],
              "strict": [Function],
              "style": [Function],
              "target": [Function],
              "to": [Function],
            },
            "render": [Function],
          }
        }
        to="/masternodes/create"
      >
        <MdAdd />
        <span
          className="d-lg-inline"
        >
          Create
        </span>
      </Button>
    </ButtonGroup>
    <SearchBar
      onChange={[Function]}
      searching={false}
      toggleSearch={[Function]}
    />
  </header>
  <div
    className="content"
  >
    <section>
      <Connect(MasternodesList)
        history={
          Object {
            "action": "POP",
            "block": [Function],
            "canGo": [Function],
            "createHref": [Function],
            "entries": Array [
              Object {
                "hash": "",
                "pathname": "/",
                "search": "",
                "state": undefined,
              },
            ],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "index": 0,
            "length": 1,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "pathname": "/",
              "search": "",
              "state": undefined,
            },
            "push": [Function],
            "replace": [Function],
          }
        }
        searchQuery=""
      />
    </section>
  </div>
</div>
`;
